### PR TITLE
Included all changes that were not assigned to Alex

### PR DESCRIPTION
**Pull request by reneehlozek**
Mainly rewriting to address issues of clarity, typos and sentence construction.

### CODE REVIEW

**Comment by reneehlozek** [comment]
I'm going to leave it to you @aimalz to resolve the conflicts.

**Comment by reneehlozek** [comment]
I am still working on the results section to make sure that our points are clear, but have made changes to the text in the previous sections, including responses to comments from Tina on language and interpretability. Please comment on this PR @aimalz in particular with reference to removing symbols - we want to make sure that folks outside of only the stats community read and engage with this text! I will make changes to the results section tomorrow morning, so please let me know the status of this PR then.

File: paper/tex/methods.tex [hunk 1]
```diff
@@ -14,8 +14,7 @@ \section{Methods}
 To optimally discriminate between classification techniques, there must be a performance metric, a single scalar value quantifying how appropriate a classifier is for the task at hand.
 Choosing a metric for \plasticc\ therefore is logically entwined with the challenge goals.
 The goal of \snphotcc\ was to identify good classifiers of SN Ia used for a photometric cosmology analysis, which lends itself to a metric that optimizes the purity of any resultant data set.
```
Comment (aimalz) [inline paper/tex/methods.tex]: Bizarrely, this isn't showing up correctly in the diff, but if you look at line 16 of methods.tex in both branches, the one you're proposing conflicts with @RickKessler's feedback about SNPhotCC, which was why the text in issue/65/2nddraftfeedback was changed the way it was.

**Comment by aimalz** [comment]
@reneehlozek Right on!  As I've said before, the intended audience is _only_ astronomers, not statisticians, and, as Tom has pointed out, I clearly don't even read statistics papers, so this language doesn't come from there -- the corrections and clarifications are appreciated!

I took a look at the diff and incorporated many of the smaller typographical/linguistic changes.  However, your modifications are quite extensive, too much for me to review given that there's a bug in GitHub preventing a proper comparison, even if I didn't have other commitments this week.  It would at least have helped if @tinapeters's comments had been made into an issue so I could try to make sense of it without dedicating time I no longer have -- I didn't even know she was trying to give me feedback!  

If so many changes were requested, do you think we should check with the Pub. Board, since we told them the last version of (189596d, I think?) was the final version?  I don't want them to think we're going to submit a different paper after getting their approval.

**Comment by reneehlozek** [comment]
Most of the major changes so far are related to rewriting the text for smoothness and clarity (and making it more readable) - a significant number of them are from a week ago. I'm happy to resolve these conflicts - I'm only waiting as you asked me to. I particularly asked @tinapeters to have a read of the paper for clarity given that we've had comments that it is a difficult paper to understand.

File: paper/tex/discussion.tex [hunk 2]
```diff
@@ -1,38 +1,44 @@
...
 
-Supernova cosmology with spectroscopically confirmed lightcurves benefits from true positives, which contribute to the constraining power of the analysis by including one more data point; when the class we're interested in is as plentiful as SN Ia and our resources limited a priori, we may not be concerned by a high rate of false negatives.
+Even within the scope of spectroscopic follow-up as a primary motivation for early light curve classification, the goals of model-fitting to known classes and discovery of new classes would likely not share an optimally metric. The critical question for choosing the most appropriate metric for any specific science goal motivating follow-up observations is to maximize information.
+
+We provid two examples of cases of the kind of new information one needs to maximize from classification, and the qualities of a deterministic metric might enable it.
```
Comment (sethdigel) [inline paper/tex/discussion.tex]: provid -> provide

**Pull request closed without merging**